packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -413,6 +413,9 @@ function NoteEditor(props: NoteEditorProps) {
 		fontSize: Setting.value('style.editor.fontSize'),
 		contentMaxWidth: props.contentMaxWidth,
 		isSafeMode: props.isSafeMode,
+		// We need it to identify the context for which media is rendered.
+		// It is currently used to remember pdf scroll position for each attacments of each note uniquely.
+		mediaParentId: props.noteId,
```

Comment:
That's a leaky abstraction - that component doesn't need to know about a mediaParentId. Just pass the note ID or whatever is needed at the point it's needed.